lib/open_project/configuration.rb

```diff
@@ -189,15 +189,19 @@ def with(settings)
       end
 
       def configure_cache(application_config)
-        return unless @config['rails_cache_store']
+        # return if cache store is set and there is nothing to overwrite it
+        return if application_config.cache_store.present? \
```

Comment:
Seeing as you had to comment this it would lend itself to be extracted into a method with a name describing what it does. E.g. `#override_cache_config?`.